Riot/Assets/en.lproj/Vector.strings

```diff
@@ -507,6 +507,8 @@ Tap the + to start adding people.";
 "threads_beta_information_link" = "Learn more";
 "threads_beta_enable" = "Try it out";
 "threads_beta_cancel" = "Not now";
+"threads_discourage_information_1" = "Your homeserver does not currently support threads, so this feature may be unreliable. Some threaded messages may not be reliably available. ";
```

Comment:
Possibly:
```suggestion
"threads_discourage_information_1" = "Your homeserver does not currently support threads, so this feature may be unreliable. Some threaded messages may not be reliably available.";
```

Reply:
Thanks for the suggestion but i'd like to keep the space between the text and the link.

Reply:
Oh derp, I missed that there was a link after this, ignore me! 🤐